internal/rpchelp/helpdescs_en_US.go

```diff
@@ -0,0 +1,399 @@
+	"infowalletresult-timeoffset":      "The time offset",
+	"infowalletresult-connections":     "The number of connected peers",
+	"infowalletresult-proxy":           "The proxy used by the server",
+	"infowalletresult-difficulty":      "The current target difficulty",
+	"infowalletresult-testnet":         "Whether or not server is using testnet",
+	"infowalletresult-relayfee":        "The minimum relay fee for non-free transactions in BTC/KB",
+	"infowalletresult-errors":          "Any current errors",
+	"infowalletresult-paytxfee":        "The increment used each time more fee is required for an authored transaction",
+	"infowalletresult-balance":         "The balance of all accounts calculated with one block confirmation",
+	"infowalletresult-walletversion":   "The version of the address manager database",
+	"infowalletresult-unlocked_until":  "Unset",
+	"infowalletresult-keypoolsize":     "Unset",
+	"infowalletresult-keypoololdest":   "Unset",
+
+	// GetNewAddressCmd help.
+	"getnewaddress--synopsis": "Generates and returns a new payment address.",
+	"getnewaddress-account":   "Account name the new address will belong to",
+	"getnewaddress--result0":  "The payment address",
+
+	// GetRawChangeAddressCmd help.
+	"getrawchangeaddress--synopsis": "Generates and returns a new internal payment address for use as a change address in raw transactions.",
+	"getrawchangeaddress-account":   "Account name the new internal address will belong to",
+	"getrawchangeaddress--result0":  "The internal payment address",
+
+	// GetReceivedByAccountCmd help.
+	"getreceivedbyaccount--synopsis": "Returns the total amount received by addresses of some account, including spent outputs.",
+	"getreceivedbyaccount-account":   "Account name to query total received amount for",
+	"getreceivedbyaccount-minconf":   "Minimum number of block confirmations required before an output's value is included in the total",
+	"getreceivedbyaccount--result0":  "The total received amount valued in bitcoin",
+
+	// GetReceivedByAddressCmd help.
+	"getreceivedbyaddress--synopsis": "Returns the total amount received by a single address, including spent outputs.",
+	"getreceivedbyaddress-address":   "Payment address which received outputs to include in total",
+	"getreceivedbyaddress-minconf":   "Minimum number of block confirmations required before an output's value is included in the total",
+	"getreceivedbyaddress--result0":  "The total received amount valued in bitcoin",
+
+	// GetTransactionCmd help.
+	"gettransaction--synopsis":        "Returns a JSON object with details regarding a transaction relevant to this wallet.",
+	"gettransaction-txid":             "Hash of the transaction to query",
+	"gettransaction-includewatchonly": "Also consider transactions involving watched addresses",
+
+	// HelpCmd help.
+	"help--synopsis":   "Returns one line summaries for every server method or detailed usage of a single request.",
```

Comment:
The help command can do individual commands too.  This is what btcd says:

`Returns a list of all commands or help for a specified command`

Probably should make then consistent.